docs/src/components/Sandpack.jsx

```diff
@@ -112,8 +112,8 @@ export default function ComponentSandpack({
     './styles.css',
     ...Object.keys(dependencies),
     ...Object.keys(files).reduce((importPaths, fileAbsPath) => {
-      // Only automatically import .css or .js files for now
-      if (fileAbsPath.endsWith('.css') | fileAbsPath.endsWith('.js')) {
+      // Only automatically import .css, .js, or .mjs files for now
+      if (fileAbsPath.endsWith('.css') | fileAbsPath.endsWith('.js') | fileAbsPath.endsWith('.mjs')) {
```

Comment:
discussed internally, if possible we should avoid renaming all ESM dist files to have the `.mjs` extension to not have breaking changes. using named exports internally is a good workaround.


https://github.com/muxinc/media-chrome/compare/main...luwes:dist?expand=1
https://esbuild.github.io/content-types/#default-interop

it’s recommend for library authors by esbuild

> If you are a library author: When writing new code, you should strongly consider avoiding the default export entirely. It has unfortunately been tainted with compatibility problems and using it will likely cause problems for your users at some point.